MeshLib/Properties-impl.h

```diff
@@ -128,3 +128,32 @@ PropertyVector<T>* Properties::getPropertyVector(std::string const& name)
     return dynamic_cast<PropertyVector<T>*>(it->second);
 }
 
+template <typename T>
+PropertyVector<T> const* Properties::getNodalNComponentPropertyVector(
```

Comment:
Would be nice to generalize to `getPropertyVector(name, item_type, number_components)` like it is described in your documentation of the method.

Reply:
Sounds good. I added the generalization.